packages/analytics-js-common/src/utilities/checks.ts

```diff
@@ -64,6 +64,13 @@ const isDefinedNotNullAndNotEmptyString = (value: any): boolean =>
  */
 const isTypeOfError = (obj: any): obj is Error => obj instanceof Error;
 
+/**
+ * Determines if the SDK is running inside a chrome extension
+ * @returns boolean
+ */
+const isSDKRunningInChromeExtension = (): boolean =>
```

Comment:
I'd make this a state entry like `appEnvironment`. Maybe in the future, we can also collect information like the framework used, etc.

Reply:
CC @bardisg

Reply:
Lets see if we need something like that as part of the JS metrics project